Automattic/jetpack
# Blogroll: wire site recommendations endpoint

This PR wires the site recommendation endpoint.
This endpoint will eventually be consumed by the blogroll block to create the placeholder content.

p9Jlb4-8nF-p2#adding-the-block





Fixes #
https://github.com/Automattic/wp-calypso/issues/80911

Related diff
D120382-code

## Proposed changes:

* Add WPCOM_REST_API_V2_Endpoint_Following_Recommendations
* Add useRecommendations hook

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Use the useRecommendations hook in the edit file of the blogroll block.
* Verify that the correct endpoint and results are returned.
*
